web/packages/test/scripts/deploy-ethereum.sh

```diff
@@ -87,8 +91,12 @@ deploy_local() {
     echo "Waiting for geth API to be ready"
     sleep 3
 
-    if [ "$eth_fast_mode" == "true" ]; then
-        hack_beacon_client
+    # if were are running locally speed up the seconds per slot from 12 seconds to 2 seconds. if we are not
+    # running locally, revert
+    if [ "$eth_network" == "localhost" ]; then
+        set_slot_time 12 2
```

Comment:
Maybe we can further decrease the slot time to 1 second, then a sync period will be 2.27 hours.  
Anyway, it seems still a bit challenging to test the switchover process.

Reply:
I would be careful being too aggressive, as it may mean the beacon-light-client could struggle to import headers at such a fast rate. May need relayer updates too.

Reply:
It looks like 1 second per slot is fine for our local setup. The bridge hub parachain sometimes drops imports but then recovers. I think that's okay, for the sake of faster tests? Otherwise we can change the `SECONDS_PER_SLOT` back to 2 seconds.